components/LikeCoinAmount.vue

```diff
@@ -17,14 +17,16 @@
     </div>
 
     <div v-if="linkHref && linkText" class="links">
-      <a
-        :href="linkHref"
-        target="_blank"
-        rel="noopener noreferrer">
-        <material-button class="link what">
-            {{ linkText }}
-        </material-button>
-      </a>
+      <md-button
+        class="link what md-likecoin">
+        <a
+          class="lc-color-white"
+          :href="linkHref"
+          target="_blank"
+          rel="noopener noreferrer">
+          {{ linkText }}
+        </a>
+      </md-button>
```

Comment:
wrapping `<a>` with `<button` on the other hand will cause confusion, when the button is clicked but <a> is not triggered
better use @click i suppose

Reply:
how about `a > button`, since we just need the style of the `<md-button />` and `<a />` will trigger the url tooltip at the bottom left corner when use hover it. We have lots of use cases `<nuxt-link />` wrapping `<md-button />`

Reply:
just `a > button` then, but i suspect it somehow violate html spec, and `<nuxt-link />` wrappin `button` sounds like some quick hack left behind

Reply:
We found out that md-button has `href` props...